main.py

```diff
@@ -361,19 +364,43 @@ def __init__(self, *args):
 		self.restartButton.clicked.connect(self.onRestartButtonClicked)
 		self.actionExit.triggered.connect(self.onExitClicked)
 		self.actionSettings.triggered.connect(self.onSettingsClicked)
+		self.inst._p.readyReadStandardOutput.connect(self.onReadyReadStandardOutput)
+		self.inst._p.readyReadStandardError.connect(self.onReadyReadStandardError)
+		self.inst._p.stateChanged.connect(self.onXqemuStateChanged)
+
+	def onReadyReadStandardOutput(self):
+		text = self.inst._p.readAllStandardOutput().data().decode()
+		self.log.appendPlainText(text)
+
+	def onReadyReadStandardError(self):
+		text = self.inst._p.readAllStandardError().data().decode()
+		self.log.insertPlainText(text)
```

Comment:
According to docs, this adds text at current cursor position. Would a user selecting text mean that text is added in wrong location?

Reply:
You're correct (just confirmed it by testing). The recommendation where I took that from actually set the cursor before and after, so I changed the code to do that as well. It has the downside of resetting a selection the user made whenever log output is added, but it seems like this is the only way how the additional whitespaces can be avoided.

Reply:
I'm sure there's a better way. We should create an issue after merge, so we can improve this later.